deploy-web/src/utils/sdl/sdlGenerator.ts

```diff
@@ -32,10 +35,23 @@ export const generateSdl = (formData: SdlBuilderFormValues) => {
         const to = e.to.map(to => ({ ["service"]: to.value }));
         _expose["to"] = [
           {
-            global: !!e.global
+            global: !!e.global,
+            ...(e.ipName && { ip: e.ipName })
```

Comment:
Not sure if `ipName` can be `undefined` here, but if it does the spread operator would break
```suggestion
            ...(e.ipName ? { ip: e.ipName } : {})
```